examples/lighting-app/efr32/BUILD.gn

```diff
@@ -82,7 +82,8 @@ if (chip_enable_wifi) {
 }
 
 # BRD4166A --> ThunderBoard Sense 2 (No LCD)
-if (efr32_board == "BRD4166A" || efr32_board == "BRD4180A") {
+if (efr32_board == "BRD4166A" || efr32_board == "BRD4180A" ||
+    efr32_board == "BRD4186A" || efr32_board == "BRD4187A") {
```

Comment:
Why are you removing LCD support for mg24 boards?

Reply:
There was issue with WLAN connectivity and after disabling LCD (suggested by Narayan) it started working.